README.md

```diff
@@ -39,6 +39,7 @@ This extension contributes the following settings:
 | `queueCompleteMessageType` | `status` | Choose how to be notified on queue completion. Either `status`, or `message` for a permanent reminder. |
 | `statusMessageColor` | `notification.`<br>`infoBackground` | Choose the colour of the queue completion status message. |
 | `queueWatchedFiles` | `false` | When set to `true`, Push will queue watched files with changes detected instead of immediately uploading them. |
+| `autoUploadQueue` | `false` | When set to `true`, Push will automatically upload files that enter the queue. This allows for changes within VS Code to be committed on save, while not committing changes from outside VS Code (like a watcher would). |
```

Comment:
@ephemerant Please use "uploaded/uploading" here instead of "comitted/comitting". (Unless I'm missing your intention here?)